src/components/cards/IssuesCard.vue

```diff
@@ -24,13 +21,14 @@ export default {
       default: 0
     }
   },
-  computed: {
-    last() {
-      return head(this.issues);
-    }
-  },
   components: {
     Issue
+  },
+  methods: {
+    getIssues: function() {
```

Comment:
You can keep the computed property and use `slice` this way: `slice(this.issues, 0, 5)`